src/logic/calculate.js

```diff
@@ -2,31 +2,42 @@ import operate from './operate';
 
 const calculate = (data, btn) => {
   let { total, next, operation } = data;
-  const operators = ['+', '&times;', '&minus;', '÷'];
+  const operators = ['+', 'x', '-', '÷', '%'];
```

Comment:
**we need to work on the operator `÷ `**
Division by zero is not handled properly yet. Let's fix this.
The reason is that when a division by zero happens, the app crashes.

Steps to reproduce:

Divide a number by zero (e.g. 4 / 0)
The app crashes.